test/integration_test.go

```diff
@@ -107,6 +107,85 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+func TestTerragruntDownloadDir(t *testing.T) {
```

Comment:
I forgot to mention this in the previous comment about removing `t.Parallel`, but can you add a comment here explaining why we don't run this in parallel? Specifically that it is manipulating environment variables as part of the test which edits global state?

Reply:
done 👍